userspace/libsinsp/test/sinsp_utils.ut.cpp

```diff
@@ -17,13 +17,19 @@ limitations under the License.
 */
 
 #include <gtest/gtest.h>
-#include <libsinsp/utils.h>
+#include "utils.h"
```

Comment:
```suggestion
#include <libsinsp/utils.h>
```

This shouldn't be reverted. See https://github.com/falcosecurity/libs/commit/6f8cb4707ec1cde1d0acb52501367bdc39b32afa#diff-362102dc7b939ad80b6a97c6e36cb1c610f314172ebb33b9b1483b062e53c81d

cc @therealbobo

Reply:
We should really add a CI to avoid them slipping in :/

Reply:
Done.